test/inductor/test_padding.py

```diff
@@ -213,9 +215,13 @@ def f(**inputs):
         f_bad_shape, inputs_bad_shape = create_model(30522)
 
         print("benchmark for good shape")
-        latency_good_shape = do_bench(lambda: f_good_shape(**inputs_good_shape))
+        latency_good_shape = benchmarker.benchmark(
```

Comment:
Kind of a pervasive comment - this diff changes `do_bench` to `benchmarker.benchmark`, which means there are a bunch of mechanical changes to slog through without an actual API change.  Can we keep `do_bench` and just internally have it call `benchmarker.benchmark`?

Reply:
Sure, I'll add this for now (but might clean it up in the future)